### PR TITLE
Deprecate "maxDuration" documentation as the prop is removed from "<Suspense>" #22

### PR DESCRIPTION
**Pull request by dance2die**
A PR for issue #22

### CODE REVIEW

File: apis/react-suspense.md [hunk 1]
```diff
@@ -25,13 +25,10 @@ that also supports key-based invalidation and preloading but they are not strict
 
 Caches should be idempotent and should **throw promises** to resolve data fetches.
 
-## maxDuration is NOT actual Duration
+~~## maxDuration is NOT actual Duration~~
```
Comment (dance2die) [inline apis/react-suspense.md]: Would it be better if removed instead of strike-through?

File: apis/react-suspense.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ React Suspense is a generic way for components to suspend rendering while they l
   - showing a **fallback** UI if the duration of the suspense exceeds a threshold
   - **resuming** render when the fetch is fulfilled and the cache read obtains a value
 - Render is only complete and committed to the DOM when either:
-  - `maxDuration` is exceeded and the **fallback** UI is shown. React ships with built in maxDuration expiry limits (up to 5 seconds, iirc), and will use the smaller of the builtin limit or of whatever you specify (see below).
+  - a hueristic and different mechanism decides that a **fallback** UI is shown. React ships with [hard-coded built-in expiry limit of 150ms](https://github.com/facebook/react/pull/15272#issue-265972492).
```
Comment (arianon) [inline apis/react-suspense.md]: Not true anymore, https://github.com/facebook/react/pull/15367

Comment (dance2die) [reply apis/react-suspense.md]: Thank you, @arianon.
You are right (and "fresh" with the information).

`150ms` is deleted - https://github.com/facebook/react/pull/15367/files#diff-a409dc1b2c8ece1cc1fa28fe42b481ceL1829
and uses a hueristic (Just Noticeable Difference (JND) boundary) only to compute the timeout duration.

I will update the documentation and update the PR.

**Comment by dance2die** [comment]
Is Docz not used any more? It looks like failed PRs have been merged 🤔 

docz failed merges

File: apis/react-suspense.md (hunk 1 above)
Comment (sw-yx) [reply apis/react-suspense.md]: probably just remove it

**Comment by sw-yx** [comment]
docz is inactive. i think i should remove it.

anyway thanks for your PR!! great work!!

File: apis/react-suspense.md (hunk 1 above)
Comment (dance2die) [reply apis/react-suspense.md]: Thanks for the review. I removed it.

File: apis/react-suspense.md (hunk 2 above)
Comment (dance2die) [reply apis/react-suspense.md]: Thanks @arianon. I've updated the documentation & the PR.

**Comment by dance2die** [comment]
> docz is inactive. i think i should remove it.
> 
> anyway thanks for your PR!! great work!!

Ah, that's a relief 😌. Thought I messed up.

You're welcome and thank you for the fresh contents.

**Pull request merged**